apps/src/p5lab/constants.js

```diff
@@ -11,6 +11,9 @@ export const P5LabInterfaceMode = utils.makeEnum(
 /** @enum {string} */
 export const P5LabType = utils.makeEnum('GAMELAB', 'SPRITELAB', 'POETRY');
 
+/** @enum {string} */
+export const CURRENT_ANIMATION_TYPE = utils.makeEnum('default', 'background');
```

Comment:
naming nit: enums are usually cased like `P5LabType` above. i don't think we have a convention around the enum values themselves (`CurrentAnimationType.default` vs `CurrentAnimationType.DEFAULT`), but i prefer the way you have it here. sorry, i should've noticed this in my initial review!

Reply:
Happy to change it. I actually based the casing on `PICKER_TYPE` which was defined here: https://github.com/code-dot-org/code-dot-org/blob/c1d71198471e66ee23384b6479815328d39efa85/apps/src/p5lab/AnimationPicker/AnimationPicker.jsx#L27
I didn't notice the difference with `P5LabType` when I moved the new one into constants.

I also might remove this entirely and in favor of using `P5LabInterfaceMode` (defined above) for both purposes.